test_app/tests/authentication/utils/test_claims.py

```diff
@@ -5,88 +5,233 @@
 
 from ansible_base.authentication.models import AuthenticatorUser
 from ansible_base.authentication.utils import claims
+from test_app.tests.authentication.conftest import SYSTEM_ROLE_NAME
 
 
-@pytest.mark.xfail(connection.vendor == 'postgresql', reason='Test only reliable in sqlite3 due to pk references')
 @pytest.mark.parametrize(
```

Comment:
Should we add a label/comment to the various tests in here to give someone a better idea of what each is doing?

Reply:
added